rtengine/rawimage.cc

```diff
@@ -560,6 +560,13 @@ int RawImage::loadRaw(bool loadData, unsigned int imageNum, bool closeFile, Prog
                 raw_crop_cc = true;
                 int lm, tm, w, h;
                 cc->get_rawCrop(raw_width, raw_height, lm, tm, w, h);
+                is_cropped = read_crop.complete && (raw_width - read_crop.width > is_cropped_margin || raw_height - read_crop.height > is_cropped_margin);  // RT
+                if (is_cropped){                                                                                                                            // RT
+                    left_margin = read_crop.left_margin;                                                                                                    // RT
+                    top_margin = read_crop.top_margin;                                                                                                      // RT
+                    tm = 0;                                                                                                                                 // RT
+                    lm = 0;                                                                                                                                 // RT
+                }                                                                                                                                           // RT
```

Comment:
This looks like it overrides the values in `camconst.json`, but `camconst.json` should take precedence over values read from the metadata in dcraw. There should be a way to fix the bug without touching this file.

Reply:
I added that part because the incorrect data from `camconst.json` overrode the more applicable metadata. But with your correction to `camconst.json` this part might indeed be unnecessary.

Reply:
By removing this overwrite, I'm not sure when the metadata crop would be used.

Reply:
It will be used if there is no matching raw crop in `camconst.json`. There may be some cameras which have a single raw crop defined for all resolutions even though the camera has a crop mode. We will have to fix those raw crops, either by adding the correct crops or limiting the crops to the full resolution images.